packages/quantic/cypress/integration/facets/facet/facet.cypress.ts

```diff
@@ -306,6 +385,16 @@ describe('Facet Test Suite', () => {
     });
   });
 
+  describe('with invalid sorting', () => {
```

Comment:
(just for your information) for "sort" checking, in atomic, we don't check the return call from searchAPI/or headless, because we want to focus more on how atomic displays item _ so test somehow can be independent from seachAPI/headless. If anything changes from searchAPI, headless will handle that, and from UI we shouldn't be affected.
So for example, if we use sortCriteria= 'alphanumeric',   we want to make sure that it will display properly on the UI :)

Reply:
@lvu285  I'd prefer not to replicate the index sorting logic in the tests. Anyways, it might not even be possible for all sorting options. I think we can trust the index to return facet values in the correct order. So, in addition to validating the the sort criterion is properly set on the request, I could validate that the facet renders the values in the same order as the search response. It would avoid duplicating logic, but would ensure that the facet does not reorder values.

Would that be ok with you?